app/src/main/java/org/p2p/wallet/push_notifications/api/NotificationServiceApi.kt

```diff
@@ -0,0 +1,24 @@
+package org.p2p.wallet.push_notifications.api
+
+import org.p2p.solanaj.model.types.RpcRequest2
+import org.p2p.wallet.infrastructure.network.data.CommonResponse
+import org.p2p.wallet.push_notifications.model.DeviceToken
+import org.p2p.wallet.utils.emptyString
+import retrofit2.http.Body
+import retrofit2.http.POST
+import retrofit2.http.Url
+
+interface NotificationServiceApi {
+
+    @POST
+    suspend fun sendDeviceToken(
+        @Body rpcRequest: RpcRequest2,
+        @Url url: String = emptyString()
+    ): CommonResponse<DeviceToken>
+
+    @POST
+    suspend fun deleteDeviceToken(
+        @Body rpcRequest: RpcRequest2,
+        @Url url: String = emptyString()
+    ): CommonResponse<DeviceToken>
```

Comment:
DeviceTokenResponse

Reply:
`DeviceToken` is the same model as `AccountInfo`, `TokenAccounts` and so one. Why it should be named with `Response`

Reply:
We've decided to leave naming DeviceToken as it is, without Response/Request suffix, but we shall keep in mind that it's a network model 🤔